2200017/문제2.py

```diff
@@ -0,0 +1,13 @@
+# 3456
+T = int(input()) # 테스트 케이스 수
+
+for t in range(1, T+1):
+    N = list(map(int, input().split()))
+    for i in N:
+        if N.count(i) == 1: # 개수가 1개인 경우  
+            print(f'#{t} {i}')
+            break
+
+        elif N.count(i) == 3: # 개수가 3개인 경우 
+            print(f'#{t} {i}')
+            break
```

Comment:
```suggestion
        if N.count(i) == 1 or N.count(i) == 3: 
            print(f'#{t} {i}')
            break
```
`if`문과 `elif` 문 내의 수행하는 코드가 같으므로 하나에 합쳐서 작성하면 좋을 것 같습니다!